scripts/halium

```diff
@@ -60,6 +60,20 @@ identify_boot_mode() {
 	tell_kmsg "boot mode: $BOOT_MODE"
 }
 
+identify_android_image() {
+	# Checks for the provided Android image. If it's called system.img, it
+	# should be mounted at Android's /system. If it's called android-rootfs.img,
+	# it should be mounted at Android's /.
+	# Sets $ANDROID_IMAGE_MODE to:
+	#   * "rootfs" if the image should be mounted at '/android/'
+	#   * "system" if the image should be mounted at '/android/system/'
+	#   * "unknown" if neither is found
+
+	[ -f /tmpmnt/system.img ] && ANDROID_IMAGE_MODE="system"
```

Comment:
system.img isn't always in /tmpmnt, it can also be in /halium-system/var/lib/lxc/android/, depending on $file_layout